scripts/afterSign.js

```diff
@@ -6,26 +6,30 @@ const notarizeMacBuild = async function (context) {
 
     if (!process.env.AC_USERNAME) {
```

Comment:
It would have helped me debug if there was a console error for when you don't have your own version of app-builder linked, can we test for that?

Reply:
That's a good idea, but I think testing that will be enough of a thing that I'll want to do it in a separate PR. I've filed #135 to this suggestion.